omnigibson/controllers/ik_controller.py

```diff
@@ -135,19 +137,28 @@ def limiter(target_pos: Array[float], target_quat: Array[float], control_dict: D
             else MovingAverageFilter(obs_dim=control_dim, filter_width=smoothing_filter_size)
         )
         assert mode in IK_MODES, f"Invalid ik mode specified! Valid options are: {IK_MODES}, got: {mode}"
+
+        # If mode is absolute pose, make sure command input limits / output limits are None
```

Comment:
could you add a comment explaining why they should be None?

Reply:
It doesn't make sense for the action to be rescaled, if you're assuming the input is an absolute value -- it sort of defeats the purpose if you ask for a specific absolute pose and it gets rescaled arbitrarily. This was a silent bug that was occurring where by default actions are normalized, and therefore actions are rescaled. And so the unit test with absolute_pose was failing for this reason